client/locations-list-map.js

```diff
@@ -1957,6 +1951,60 @@ const initGlobalAlert = () => {
   });
 };
 
+const initEmbedListEventListener = () => {
+  // Web toggle button
+  document.getElementById('js-ToggleListButton').addEventListener('click', toggleEmbedList);
```

Comment:
I get a JS exception on the homepage and /makers pages for this (works great on embed). @linderr do we want this button on the homepage too or just embeds? Either need to wrap this to check if it's an embed, or make it work for both

Reply:
@linderr I can implement this on the homepage too, wouldn't take a second just let me know

Reply:
@eikhunter if we don't hear from Linda by am I think just only enable on the embed page and not on the homepage and then we can get this shipped. We can always follow up on adding it to the homepage

Reply:
I've just done this for the meantime, as I believe she has been moving house and we can take a separate look at the homepage

Reply:
Great job, guys. Thanks for being considerate but also working around me! Settled and back on it today. For homepage, I think it's the right direction to head towards to support v2 mobile UX/UI design that is toggleable between list and map to make mobile use friendlier and avoid you guys having to maintain 2 code bases between desktop and mobile. However, the homepage currently has the default Google Maps full screen map view toggle. This could be redundant if we implement @eikhunter's toggle and should be removed. Will create issues